docs/site/Server.md

```diff
@@ -211,6 +211,38 @@ export async function main() {
 For a complete list of CORS options, see
 https://github.com/expressjs/cors#configuration-options.
 
+### Configure query parser
+
+`qs` is used to parse query string with the default package settings.
+
+You can override the defaults by setting your own configuration:
+
+```ts
+// Rest server config
+rest: {
+  queryParser: {...},
```

Comment:
Please fix the option here to be `expressSettings`.

Reply:
done